tools/topology/sof-imx8-processing-wm8960.m4

```diff
@@ -0,0 +1,91 @@
+#
+# Topology for i.MX8MP board with wm8960 codec
```

Comment:
Why is the topology called "processing", not "codec_adapter" or anything related?

Reply:
@paulstelian97 I think I followed the initial naming from Intel. I could change it with a better naming. Codec_adapter is more of an internal naming users should care about this.

Reply:
Maybe "compress" or something.